sci-libs/primesieve/primesieve-5.7.3.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primesieve is a program and C/C++ library that generates primes \
+using a highly optimized sieve of Eratosthenes implementation. It counts the \
+primes below 10^10 in just 0.45 seconds on an Intel Core i7-6700 CPU \
+(4 x 3.4GHz). primesieve can generate primes and prime k-tuplets up to 2^64."
+HOMEPAGE="http://primesieve.org/"
+COPYRIGHT="2010 - 2016, Kim Walisch"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/kimwalisch/primesieve/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="22c5ec34f020592cd559551e3e6e1e86aa51f6e34d639bef352d155ea415ac62"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	primesieve$secondaryArchSuffix = 7.0.3 compat >= 7
+	cmd:primesieve = 7.0.3 compat >= 7
```

Comment:
Can you please replace:
```
	primesieve$secondaryArchSuffix = 7.0.3 compat >= 7
	cmd:primesieve = 7.0.3 compat >= 7
```
by
```
	primesieve$secondaryArchSuffix = $portVersion
	cmd:primesieve = $portVersion
```
(We don't need compat for cmd items, if I'm not mistaken.)